dockstore-common/src/test/resources/fixtures/dockstore12.yml

```diff
@@ -2,6 +2,7 @@ version: 1.2
 workflows:
   -  name: foobar
      subclass: wdl
+     publish: true
```

Comment:
Add testing case if publish is missing

Reply:
See last workflow below this (one without a name). In the test, I check it for `null` publish.